docs/docs/features/studio.md

```diff
@@ -2,14 +2,20 @@
 title: ZMK Studio
 ---
 
-:::warning[Alpha Feature]
+:::warning[Beta Feature]
 
-ZMK Studio support is in alpha. Although best efforts are being made, keeping compatibility during active development is not guaranteed.
+ZMK Studio support is in beta. Although every effort has been made to provide a stable experience, you may still encounter issues during use. Please report any issues to [GitHub Issues](https://github.com/zmkfirmware/zmk-studio/issues)
 
 :::
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Accessing ZMK Studio
+
+If using Chrome/Edge, you can use ZMK Studio directly from your browser at https://zmk.studio/
+
+To use the native app for Linux, macOS, or Windows, download the appropriate file from the [latest release](https://github.com/zmkfirmware/zmk-studio/releases)
```

Comment:
Should we note the limitations of the browser vs. app here? The argument against it would be that it is pretty clear once you open the browser version.

Reply:
More than the comment here about Chrome/Edge?

Reply:
Shoulda clarified, specifically that BLE doesn't work in Windows/macOS.